README.md

```diff
@@ -542,8 +542,8 @@ and other multi-core embedded devices).
 
 [![★][connect-arango-image] connect-arango][connect-arango-url] An ArangoDB-based session store.
 
-[connect-arango-url]: https://www.npmjs.com/package/connect-arango
-[connect-arango-image]: https://badgen.net/github/stars/AlexanderArvidsson/connect-arango?label=%E2%98%85
+[connect-arango-url]: https://github.com/ryanking1809/connect-arango
```

Comment:
Please link to the package as published on nom.

Reply:
Sorry, I did not understand, what is "nom"? Is it "npm" misspelled? Or would it be to name it like how it is installed, like:

`[![★][github:ryanking1809/connect-arango] github:ryanking1809/connect-arango][github:ryanking1809/connect-arango]`

Reply:
Sorry for the typo. Yes, npm. We only accept packages published to npm on this list. Please publish the package to npm and update the PR 👍